### DISCUSSION TITLE
Chromium Web Store PR

### DISCUSSION THREAD

**Discussion by calhix** [Eloston/ungoogled-chromium#1653] [Ideas]
Is a PR planned in the future with the NeverDecaf web store? I think it's quite useful and would be nice to ensure people know when extensions require updates right from the start, though I can see this being out of scope for this project. Obviously I think it would be great if there was a way for it to be automatic and have the chrome store work out of the box like Brave but perhaps doing so wouldn't be privacy friendly?

**Comment by networkException** [comment]
I fear that adding an external extension to enable Google functionally will directly conflict with philosophy of the project, that being no dependency on Google. Installing the extension is straight forward when following the documentation, I guess in the future those steps should probably be more visible and accessible to users.

**Reply by wchen342** [reply]
I disagree on this one, the extension doesn't contact Google directly by itself. It will only activate if the webstore page is visited by users manually. It is like visiting google search manually, there is no reason to stop it. And since it is already in wiki, I don't see how it is conflict with the goal of the project.

**Reply by networkException** [reply]
Wouldn't the auto updating query Google servers?

**Reply by wchen342** [reply]
Ah right I forgot about that. But there is an option to disable that so we can easily patch that default to off then users can manually enable if they want.

**Reply by networkException** [reply]
That sounds fair to me

**Reply by wchen342** [reply]
Or better, we can pack it and disable the whole extension by default, and users can just go to `chrome://extensions` and enable it, rather than installing it themselves.

**Comment by squid-f** [comment]
Hi actually, I am trying to package it but it is not as straightforward. I cannot use Chromium policy. Master_preference file is not enough; at least, not yet. Would you have any hint to package this extension?
Thanks

**Reply by wchen342** [reply]
See https://github.com/wchen342/chromium-android-extension/blob/master/patches/Extensions/add-chromium-webstore-extension.patch. Unless it works differently for Android and Linux or they recently changed how it works.

**Reply by squid-f** [reply]
Awesome. Thanks. Forgot to mention I am on Linux (Mageia). I will give it a try.
